src/main/java/jssc/SerialPort.java

```diff
@@ -1301,13 +1301,6 @@ public void run() {
                         }
                     }
                 }
-                //Need to sleep some time
-                try {
-                    Thread.sleep(0, 100);
-                }
-                catch (Exception ex) {
-                    //Do nothing
-                }
```

Comment:
Per comment above, the likely needs to be conditional based on the availability of `poll()`.  Please adjust accordingly.

Reply:
Same thing here: I am not sure how to check specifically for `poll` availability, especially from Java, so the best thing I could think of is to check whether we're on Mac OS